### PR TITLE
Immersive headline

### PR DESCRIPTION
**Pull request by duarte**
Updates the immersive headline design.

**Before**

**After**

### CODE REVIEW

File: ArticleTemplates/articleTemplateContainerImmersive.html [hunk 1]
```diff
@@ -5,20 +5,16 @@
...
 
-        <div class="standfirst standfirst--mobile">
+        <div class="article__header-info">
+            __IMMERSIVE_SERIES__
+            <h1 class="article__headline headline selectable">
+                __TITLE__
+            </h1>
+        </div>
+
+        <div class="standfirst standfirst">
```
Comment (GHaberis) [inline ArticleTemplates/articleTemplateContainerImmersive.html]: double use of `standfirst` class here 👯

File: ArticleTemplates/assets/scss/garnett-type/_immersive.scss [hunk 2]
```diff
@@ -101,70 +100,83 @@
...
+                margin-left: auto;
+                margin-right: auto;
                 padding: {
-                    left: 0;
-                    right: 0;
+                    left: base-px(1);
+                    right: 340px;
+                }
+                width: 1200px;
+                transform: translateX(228px); //240 - 8
```
Comment (GHaberis) [inline ArticleTemplates/assets/scss/garnett-type/_immersive.scss]: 🕵️  not quite sure what the numbers represent, but 240 - 8 is `232px`. Could it be worth assigning `228` to a variable as it's referenced in a few places which I imagine would all need updating if it were to change?

Comment (duarte) [reply ArticleTemplates/assets/scss/garnett-type/_immersive.scss]: 🤣I dreamt of an 8 pt gutter on this template but immediately regretted it haha
Will update everything, thanks george!

Comment (duarte) [reply ArticleTemplates/assets/scss/garnett-type/_immersive.scss]: I've updated it to use the SASS calculation `240px - base-px(1)` instead, which I think is easier to understand — does that seem good to you?

Comment (GHaberis) [reply ArticleTemplates/assets/scss/garnett-type/_immersive.scss]: Sounds good to me

**Comment by duarte** [comment]
Thanks for reviewing George!

**Pull request merged**